core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -777,31 +620,23 @@ class WorkspaceService(
 
   // TODO - once workspace migration is complete and there are no more v1 workspaces or v1 billing projects, we can remove this https://broadworkbench.atlassian.net/browse/CA-1118
   private def maybeDeleteGoogleProject(googleProjectId: GoogleProjectId,
-                                       workspaceVersion: WorkspaceVersion,
-                                       userInfoForSam: UserInfo
+                                       workspaceVersion: WorkspaceVersion
   ): Future[Unit] =
-    if (workspaceVersion == WorkspaceVersions.V2) {
-      deleteGoogleProject(googleProjectId, userInfoForSam)
-    } else {
-      Future.successful()
-    }
+    if (workspaceVersion == WorkspaceVersions.V2) deleteGoogleProject(googleProjectId) else Future.successful()
 
-  private def deleteGoogleProject(googleProjectId: GoogleProjectId, userInfoForSam: UserInfo): Future[Unit] =
+  private def deleteGoogleProject(googleProjectId: GoogleProjectId): Future[Unit] =
     for {
-      _ <- deletePetsInProject(googleProjectId, userInfoForSam)
+      _ <- deletePetsInProject(googleProjectId)
       _ <- gcsDAO.deleteGoogleProject(googleProjectId)
-      _ <- samDAO
-        .deleteResource(SamResourceTypeNames.googleProject, googleProjectId.value, ctx.copy(userInfo = userInfoForSam))
-        .recover {
-          case regrets: RawlsExceptionWithErrorReport
-              if regrets.errorReport.statusCode == Option(StatusCodes.NotFound) =>
-            logger.info(
-              s"google-project resource ${googleProjectId.value} not found in Sam. Continuing with workspace deletion"
-            )
-        }
+      _ <- samDAO.deleteResource(SamResourceTypeNames.googleProject, googleProjectId.value, ctx).recover {
+        case regrets: RawlsExceptionWithErrorReport if regrets.errorReport.statusCode == Option(StatusCodes.NotFound) =>
+          logger.info(
+            s"google-project resource ${googleProjectId.value} not found in Sam. Continuing with workspace deletion"
+          )
+      }
     } yield ()
 
-  private def deletePetsInProject(projectName: GoogleProjectId, userInfo: UserInfo): Future[Unit] =
```

Comment:
This was already using the top level request context, so not much point passing one in.